### DISCUSSION TITLE
Lighthouses also as nodes?

### DISCUSSION THREAD

**Discussion by djarbz** [slackhq/nebula#945] [Q&A] [answered]
I have 2 VPS' and I would like to run a LH on each for redundancy.

I would also like these hosts to communicate with each other over the Nebula mesh, but I can't add static hosts to a lighthouse.

Is there a recommended way to do this, or will I need to run Nebula as a LH and as a Node on the same host?

**Comment by nbrownus** [comment]
The common way to do this is to add static host entries for the lighthouses just like you would for the non lighthouse nodes.

**Comment by djarbz** [comment]
Just to confirm, you are referring to this section of the configuration?
```yaml
# The static host map defines a set of hosts with fixed IP addresses on the internet (or any network).
# A host can have multiple fixed IP addresses defined here, and nebula will try each when establishing a tunnel.
# The syntax is:
#   "{nebula ip}": ["{routable ip/dns name}:{routable port}"]
# Example, if your lighthouse has the nebula IP of 192.168.100.1 and has the real ip address of 100.64.22.11 and runs on port 4242:
static_host_map:
  "10.0.0.1": ["123.237.128.56:143"]
```

**Reply by jasikpark** [reply, marked as answer]
Yes, any regular host needs a list of static addresses of lighthouses to talk to, https://nebula.defined.net/docs/config/static-host-map/.

So any lighthouse that should also be a regular host should have the other lighthouses in its static_host_map

**Reply by djarbz** [reply]
Excellent, thank you!
